README.md

```diff
@@ -499,6 +500,9 @@ SAAS
   * IAAS
   * PAAS
   * SAAS</summary><br><b>
+  * IAAS - Users have control over complete Operating System and need not to worry about the physical resources, which is managed by Cloud Service Provider.
+  * PAAS - CLoud Service Provider takes care of Operating System, Middlewares and users only need to focus on our Data and Application.
+  * SAAS - Users only worry about the code and the Application, Data, OS and all stuffs are managed by Cloud Service Provider.
```

Comment:
Not accurate. Data, in any of the models, can reside locally or be managed by the cloud provider